Makefile

```diff
@@ -93,6 +93,10 @@ testing-project-ci:  ## Creates for all yml files in ./test_project_values a tes
 		make testing-project-ci-single VALUES_FILE=$$VALUES; \
 	done
 
+svu: ## Creates a new version, args options major, minor & patch
+	@go run github.com/caarlos0/svu $(args) --strip-prefix > config/version.txt
+	@go run github.com/caarlos0/svu $(args)
+
```

Comment:
We already have `make release`. Why is that needed?

Reply:
To bumb version in worklfow based on major, minor & patch. With this is possible to bumb version from the Github Workflow  UI.

<img width="1470" alt="Screenshot 2023-02-26 at 14 34 55" src="https://user-images.githubusercontent.com/8136842/221413669-6b827b40-275c-4808-b03a-427c427aad18.png">